pootle/apps/pootle_translationproject/models.py

```diff
@@ -158,7 +158,7 @@ class TranslationProject(models.Model, CachedTreeItem):
 
     language = models.ForeignKey(Language, db_index=True)
     project = models.ForeignKey(Project, db_index=True)
-    real_path = models.FilePathField(editable=False)
+    real_path = models.FilePathField(editable=False, null=True, blank=True)
```

Comment:
This is used in other parts of the codebase. I unsure if those other places require adjustments to handle the `None` value.